docs/setup/fivetran-setup.md

```diff
@@ -0,0 +1,27 @@
+# fivetran project setup
+
+To set up a new project in Fivetran follow these steps:
+
+1. First, ensure you have met the following pre-requisites:
+    - You have set up a Snowflake Account for the project (follow all instructions from snowflake-setup.md)
```

Comment:
Let's link to the actual doc, I think this will work:

```suggestion
    - You have set up a Snowflake Account for the project (follow all instructions from [here](./snowflake-setup.md))
```